cli/index.js

```diff
@@ -0,0 +1,47 @@
+import fs from "fs";
+import path from "path";
+import { extendCradle } from "./extend-cradle.js";
+import { generateSchema } from "./shema-generator.js";
+import { generateModule } from "./module-generator.js";
+// import { generateRepository } from "./repository-generator.js";
+import { extendDIResolvers } from "./di-resolvers-generator.js";
+
+const moduleName = process.argv[2];
+
+if (!moduleName) {
+    console.error(
+        "❌ Please provide a module name. Example: npm run generate-module authCustomModule"
+    );
+
+    process.exit(1);
+}
+
+const nameCamel = moduleName.charAt(0).toLowerCase() + moduleName.slice(1);
+const namePascal = moduleName.charAt(0).toUpperCase() + moduleName.slice(1);
+
+const nameKebab = moduleName
+    .replace(/([a-z0-9])([A-Z])/g, "$1-$2")
+    .toLowerCase();
+
+const modulePath = path.join(process.cwd(), "src/modules", nameCamel);
```

Comment:
Please update it to be kebab case. Since directories follow the same principle as file names.